app/controllers/api/v1/site_controller.rb

```diff
@@ -127,6 +130,9 @@ def view_study
           response 406 do
             key :description, 'Accept or Content-Type headers missing or misconfigured'
           end
+          response 410 do
+            key :description, 'Study workspace is not found, cannot complete action'
```

Comment:
The string "Study workspace is not found, cannot complete action" is duplicated some 23 times in this PR.  That seems like a lurking maintainability issue -- a developer could easily change the string in one place, but overlook changing it in another.  

Would it be worth extracting the message into a separate file and referencing it here by a variable, to ease future updates?

We've discussed ideas for DRY in the controller code previously; some related ideas are e.g. [here](https://github.com/broadinstitute/single_cell_portal_core/pull/201#discussion_r248796792) and [here](https://github.com/broadinstitute/single_cell_portal_core/pull/172#discussion_r222387268).

Reply:
I agree. I think this should be saved somehow as a variable.

Reply:
I think it's worth noting that is only the "display" when looking at the swagger UI.  This is not the response returned by the API itself.  Having said that, you're right - this is very much not DRY.  I'll make a commit to extract this out somehow, and put in a framework for handling common response descriptions like this.